src/W3C.xml

```diff
@@ -0,0 +1,58 @@
+<SPDX name="W3C Software Notice and License (2002-12-31)" identifier="W3C" osi-approved="true">
+  <urls>
+    <url>http://www.w3.org/Consortium/Legal/2002/copyright-software-20021231.html</url>
+    <url>http://www.opensource.org/licenses/W3C</url>
+  </urls>
+  <notes>This license was released: 13 December 2002. The standard header is provided via a link from the original
+         license page.</notes>
+  <header>Copyright (C) 
+    <alt name="year" match=".+">[$date-of-software]</alt> World Wide Web Consortium, (Massachusetts Institute of
+    Technology, European Research Consortium for Informatics and Mathematics, Keio University). All Rights Reserved.
+    This work is distributed under the W3C® Software License in the hope that it will be useful, but WITHOUT ANY
+    WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+  </header>
+  <license>
+    <title>
+      <p>W3C SOFTWARE NOTICE AND LICENSE</p>
+    </title>
+    <optional>
+      <p>This work (and included software, documentation such as READMEs, or other related items) is being
+         provided by the copyright holders under the following license.</p>
+      <p>License</p>
+      <p>By obtaining, using and/or copying this work, you (the licensee) agree that you have read, understood,
```

Comment:
The body of this license appears to start at line 22 with the sentence "By obtaining..."